src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.model;
+
+import static racingcar.utlis.RandomNumberGenerator.*;
+
+import racingcar.validator.NameValidator;
+
+public class Car {
+
+	static final int MOVE_CRITERIA = 4;
+	private final String name;
+	private int position;
+
+	public Car(String name, int position) {
+		NameValidator.checkCarName(name);
+		this.name = name;
+		this.position = position;
+	}
+
+	public void decideMove() {
+		if (isMove()) {
+			moveCar();
+		}
+	}
+
+	private boolean isMove() {
+		return getRandomInt() >= MOVE_CRITERIA;
+	}
+
+	private void moveCar() {
```

Comment:
Car.moveCar `Car`가 중복되는 것 같아요.
```suggestion
	private void move() {
```